### ISSUE TITLE
[future]not specification requires target

### ISSUE THREAD

**Issue by vaaaaanquish** [m3dev/gokart#179]
```
class foo(gokart.TaskOnKart):
    target = gokart.TaskInstancePrameter()

    def run(self):
        self.load_data_frame(required_columns={'foo'})    # AttributeError: 'dict' object has no attribute 'empty'
```

When the length of the dict is 1, what to load of require task.

**Comment by Hi-king** [comment]
I think `dict` output cannot be used with `load_data_frame`.
With `load_data_frame` keys of dict will be lost.

**Comment by Hi-king** [comment]
OK, I've realized point is that you need shorthand of `self.load_data_frame("target")` with `self.load_data_frame()`

**Comment by vaaaaanquish** [comment]
merged #180
